package.json

```diff
@@ -66,27 +66,47 @@
     ]
   },
   "devDependencies": {
+    "@angular/animations": "9.1.2",
+    "@angular/cdk": "9.2.1",
+    "@angular/common": "9.1.2",
     "@angular/compiler": "9.1.2",
     "@angular/compiler-cli": "9.1.2",
+    "@angular/core": "9.1.2",
+    "@angular/forms": "9.1.2",
+    "@angular/localize": "9.1.2",
+    "@angular/material": "9.2.1",
+    "@angular/platform-browser": "9.1.2",
+    "@angular/platform-browser-dynamic": "9.1.2",
+    "@angular/platform-server": "9.1.2",
+    "@angular/router": "9.1.2",
+    "@angular/service-worker": "9.1.2",
+    "@babel/core": "7.9.0",
+    "@babel/generator": "7.9.4",
+    "@babel/preset-env": "7.9.0",
+    "@babel/template": "7.8.6",
+    "@jsdevtools/coverage-istanbul-loader": "3.0.3",
     "@bazel/bazel": "2.1.0",
```

Comment:
nit: switch the bazelisk?

Reply:
latest version has a `bazel` bin entry as an alias so `bazel` can still be used in package.json scripts

Reply:
Switched over.